openmetadata-ui/src/main/resources/ui/playwright/support/entity/ingestion/ServiceBaseClass.ts

```diff
@@ -262,8 +262,8 @@ class ServiceBaseClass {
           intervals: [30_000, 15_000, 5_000],
         }
       )
-      // To allow partial success
-      .toContain('success');
```

Comment:
same here

Reply:
It's contain so partialSucess will work too right